### PR TITLE
Gems Rubocop and Travis + Fixes to offenses by rubocop

### PR DESCRIPTION
**Pull request by antuaPeralta**
## Description

- Added Rubocop to analyze the code
- Added Travis to test and deploy
- Fixes made for rubocop offenses

## Trello Card
https://trello.com/c/qUhXbATg/8-configurar-rubocop
https://trello.com/c/pzlloWP1/12-configurar-travisci

## Screenshots
Captura de Pantalla 2022-06-07 a la(s) 11 53 10 a m

### CODE REVIEW

**Comment by truccomarcos** [comment]
LGTM 👍  pls wait for @ranuccimariano to review it as well

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: ruby
+rvm:
+ - 3.1.2
```
Comment (ranuccimariano) [inline .travis.yml]: Missing empty line at the end of file

File: Gemfile [hunk 2]
```diff
@@ -1,72 +1,80 @@
-source "https://rubygems.org"
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+source 'https://rubygems.org'
+git_source(:github) { |repo| `https://github.com/#{repo}.git` }
```
Comment (ranuccimariano) [inline Gemfile]: Was this change recommended bay Rubocop 🤔 ? We commonly use double quotes for string interpolation

File: .gitignore [hunk 3]
```diff
@@ -29,3 +29,4 @@ test
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+Gemfile.lock
```
Comment (ranuccimariano) [inline .gitignore]: We usually don't add the `Gemfile.lock` to the `.gitignore` file

**Review by ranuccimariano — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (antuaPeralta) [reply Gemfile]: Captura de Pantalla 2022-06-07 a la(s) 2 33 00 p m

Yes, I saw it strange but that's what rubocop said

**Review by ranuccimariano — APPROVED** [review]
👍

**Pull request merged**